packages/twenty-front/src/modules/object-record/hooks/useCreateManyRecords.ts

```diff
@@ -141,6 +146,7 @@ export const useCreateManyRecords = <
         throw error;
       });
 
+    await refetchAggregateQueries();
```

Comment:
this will seek to refetch aggregate queries even if there were no AggregateQueries performed before (like on table view for instance). I think it's ok but it does trigger a warning (yellow) in the console.

Reply:
@charlesBochet fyi